libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteProgressChangeListenerTest.kt

```diff
@@ -55,6 +55,17 @@ class MapRouteProgressChangeListenerTest {
         verify(exactly = 0) { routeLine.draw(any<List<DirectionsRoute>>()) }
     }
 
+    @Test
+    fun `should not update maneuver arrow when directionsRoute is null`() {
```

Comment:
@abhishek1508 can you verify `should update maneuver arrow when visible` is still correct?

Should we be updating maneuver arrow with null geometry?